test-utils/runtime-tester/src/scenario_builder.rs

```diff
@@ -0,0 +1,70 @@
+use crate::run_test::{BlockConfig, NetworkConfig, Scenario, TransactionConfig};
+use near_crypto::{InMemorySigner, KeyType};
+use near_primitives::{
+    transaction::Action,
+    types::{AccountId, BlockHeight, Nonce},
+};
+
+use std::str::FromStr;
+
+pub struct ScenarioBuilder {
+    height: BlockHeight,
+    nonce: Nonce,
+    scenario: Scenario,
+}
+
+impl ScenarioBuilder {
+    pub fn new(num_accounts: usize, use_in_memory_store: bool) -> Self {
```

Comment:
I'd suggest the following AP:

```rust
let builder = ScenarioBuilder::new() // some default number of accounts here
  .number_of_accounts(92) // in case we want to change it
  .in_memory_store(false); // or to use a different store
```

This is a tool for testing and investigation, and here it pays off to make API frictionless.